### PR TITLE
Differentiate between node template and user cluster creation.

### PR DESCRIPTION
**Pull request by hulquest**
Change bootstrap_rancher to be rancher_create_node_template so it can be easily understood.  
Added a parameter to name a node template.

### CODE REVIEW

File: rancher.tfvars.example [hunk 1]
```diff
@@ -70,17 +70,21 @@ vm_network = "network1"
...
 # use_auto_dns_url = false
 
-# (Optional) Automatically configure rancher:
+# (Optional) Automatically configure rancher node template:
 # - Updates the default admin password, provided by the rancher_password variable.
 # - Enables telemetry 
 # - Creates cloud credentials for vCenter
 # - Creates node template for vCenter
-# bootstrap_rancher = true
+# - Set the password for the admin user
```
Comment (bgentry-ntap) [inline rancher.tfvars.example:78]: This already existed above

File: rancher.tfvars.example [hunk 2]
```diff
@@ -70,17 +70,21 @@ vm_network = "network1"
 # (Optional) Automatically set rancher_server_url URL to <node ip>.nip.io
 # use_auto_dns_url = false
 
-# (Optional) Automatically configure rancher:
+# (Optional) Automatically configure rancher node template:
 # - Updates the default admin password, provided by the rancher_password variable.
```
Comment (bgentry-ntap) [inline rancher.tfvars.example:74]: might want to mention that `rancher_password` is required

**Review by bgentry-ntap — APPROVED** [review]
We will need to update the secrets in jenkins too with the interface changes

File: rancher.tfvars.example (hunk 1 above)
Comment (hulquest) [reply rancher.tfvars.example:78]: Fixed.

File: rancher.tfvars.example (hunk 2 above)
Comment (hulquest) [reply rancher.tfvars.example:74]: Added a clarification.

File: .github/workflows/workflow.yaml [hunk 3]
```diff
@@ -149,7 +149,7 @@ jobs:
           TF_VAR_vm_network: ${{ secrets.TF_VPC_MGMT_NETWORK }}
           TF_VAR_vsphere_vm_folder: ${{ secrets.TF_VSPHERE_VM_FOLDER }}
           TF_VAR_rancher_password: ${{ secrets.TF_RANCHER_PASSWORD }}
-          TF_VAR_bootstrap_rancher: true
+          TF_VAR_rancher_create_node_template: true
```
Comment (colep-ntap) [inline .github/workflows/workflow.yaml:152]: Should we expose this option to the user?

Comment (hulquest) [reply .github/workflows/workflow.yaml:152]: Yeah, we have a card on the board to do just that. At this layer, it will remain disabled by default.

**Review by colep-ntap — APPROVED** [review]

**Pull request merged**